services/nomad/build/buildsync-aarch64.nomad

```diff
@@ -36,19 +28,34 @@ job "buildsync-aarch64" {
 
       template {
         data = <<EOF
-{{- with secret "secret/buildsync/ssh" -}}
-{{.Data.private_key}}
-{{- end -}}
-EOF
-        destination = "secrets/id_rsa"
-        perms = "0400"
-      }
+settings {
+    statusFile = "/tmp/lsyncd.status",
+    nodaemon = true,
+}
 
-      template {
-        data = <<EOF
-b-fsn-de.node.consul ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIP1ykMRsB3/DOv/NhnhajQIB99xUdM7RauBS5ptj2i4q
+sync {
+    default.rsync,
+    source = "/pkgs/aarch64",
+    target = "rsync://a-fsn-de.node.consul:{{ nomadService "build-rsyncd".Port }}/pkgs/musl",
```

Comment:
`nomadService` returns an iterable.  You will need to wrap this in a range to fetch just one since it provides a context, not a direct service object.